### PR TITLE
Modal contact Homeless

### PR DESCRIPTION
**Pull request by AllanBallester**
#76

### CODE REVIEW

File: app/views/homelesses/_modalphone.html.erb [hunk 1]
```diff
@@ -1,8 +1,6 @@
 <div class='homeless_phone' >
-<p> No one Phone registered yet. However map is now revealed.</p>
-<p> Cross <span class="type-rise"> <%= @homeless.name %> </span> according to the last 5 posiiton markers revealed on the map. Blue marker is the last. </p>
+<p> No one Phone registered yet. However the profile's map is now revealed.</p>
```
Comment (pel-daniel) [inline app/views/homelesses/_modalphone.html.erb:2]: "No one Phone registered yet" sounds strange to me. Maybe change to "No phone registered yet" ?
Is what you mean ?

File: app/views/homelesses/_modalphone.html.erb [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
+<p> You can find now until the last 5 <span class="type-rise"> <%= @homeless.name %> </span>'s position. Blue Pin show the most recent one. </p>
```
Comment (pel-daniel) [inline app/views/homelesses/_modalphone.html.erb]: I will also remove the word 'until' here.

File: app/views/homelesses/_modalwithphone.html.erb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <div class='homeless_phone' >
 <p> Phone registered. </p>
 <p> Contact <%= @homeless.name %> at <span class="type-rise"> <%= @homeless.phone %> </span> and define When and What to give. </p>
+<p> In addition you can find now until the last 5 <span class="type-rise"> <%= @homeless.name %> </span>'s position in the profile map. Blue Pin show the most recent. </p>
```
Comment (pel-daniel) [inline app/views/homelesses/_modalwithphone.html.erb]: Same here, remove the word 'until'

**Comment by AllanBallester** [comment]
#76

**Pull request merged**